internal/pkg/cli/dialog/rename_instance.go

```diff
@@ -10,9 +10,9 @@ import (
 )
 
 // AskRenameInstance - dialog to rename template instance.
-func (d *Dialogs) AskRenameInstance(projectState *project.State, opts *options.Options) (out renameOp.Options, err error) {
+func (p *Dialogs) AskRenameInstance(projectState *project.State, opts *options.Options) (out renameOp.Options, err error) {
```

Comment:
Na tomhle mi failoval linter a IDE mi tvrdilo že žádná `AskRenameInstance` neexistuje, tak jsem to přejmenoval. 🤷‍♂️😃